### ISSUE TITLE
Homepage / Detail Page - Contrast Issue

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#523]
SC 1.4.3 AA – Contrast – “Help us improve our site” has insufficient color contrast of 3.2 (foreground
color: #ffffff, background color: #28A644)

**Comment by ewlarson** [comment]
Fixed in 9e6efdb7e519c5c2841af8795e166401812a199a
